packages/help-center/jest.config.js

```diff
@@ -1,4 +1,7 @@
 module.exports = {
 	preset: '../../test/packages/jest-preset.js',
+	setupFiles: [ '<rootDir>/jestSetup.ts' ],
 	testEnvironment: 'jsdom',
+	moduleFileExtensions: [ 'ts', 'tsx', 'js', 'jsx', 'json', 'node' ],
```

Comment:
```suggestion
	moduleFileExtensions: [ 'ts', 'tsx', 'js', 'json' ],
```
From what I see these are the only extensions we need to account for in the help center.
The most common ones should be earlier in the array.

Reply:
Thanks for taking a look at this! I'll make the changes